config/locales/en.yml

```
@@ -106,7 +106,9 @@ en:
       description: Need help? Get in touch with your local electoral registration team. They can tell you if you’re on the electoral register, or if you’ve registered for a postal or proxy vote.
       title: Get help with electoral registration
     service:
-      description: For questions about your poll card, polling place, or about returning your postal voting ballot, contact your council."
+      matched_postcode_html: We've matched the postcode to <strong>%{postcode} %{electoral_service_name}</strong>.
```

Looking at the [existing code](https://github.com/alphagov/frontend/blob/48268f18f18c12ba3c33f77be551d1e59829bc9f/app/views/electoral/results.html.erb#L11) I think this should be:

```suggestion
      matched_postcode_html: We've matched the postcode <strong>%{postcode}</strong> to <strong>%{electoral_service_name}</strong>.
```

---

😱 good spot! I have changed it.

---

Thanks - by the way the change hasn't come through, maybe you've not pushed it or one of us has a wire crossed?